src/logic/safe/store/actions/transactions/utils/newTransactionsHelpers.ts

```diff
@@ -0,0 +1,11 @@
+import { Transaction, TxType } from '../../../models/types/transactions'
+
+export const isMultiSigTx = (tx: Transaction): boolean => {
+  return TxType[tx.txType] === TxType.MULTISIG_TRANSACTION
```

Comment:
nitpick: TxType[tx.type] looks weird, can we come up with a better name for this?

Reply:
what do you suggest?